cdk/lib/frontend.ts

```diff
@@ -18,5 +34,384 @@ export class Frontend extends GuStack {
         Stage: GuStageParameter.getInstance(this),
       },
     });
+
+    const app = "frontend";
+
+    const userData = `#!/bin/bash -ev
+    mkdir /etc/gu
+    aws --region ${this.region} s3 cp s3://membership-dist/${this.stack}/${this.stage}/${app}/support-frontend_1.0-SNAPSHOT_all.deb /tmp
+    dpkg -i /tmp/support-frontend_1.0-SNAPSHOT_all.deb
+    /opt/cloudwatch-logs/configure-logs application ${this.stack} ${this.stage} ${app} /var/log/support-frontend/application.log '%Y-%m-%dT%H:%M:%S,%f%z'`;
+
+    const membershipSubPromotionsTable = this.withStageDependentValue({
+      app,
+      variableName: "MembershipSubPromotions",
+      stageValues: {
+        // TODO: Should these dynamo tables be region specific?
```

Comment:
DevEx pointed out that these tables aren't region specific and that it might be preferable if they were

Reply:
By this do you mean tagged to a specific AWS region?

Reply:
Yeah that's right. I would have to defer to DevX for exactly the benefits that would give us. I think it was that keeping everything region specific would isolate our infrastructure to just a single region. This would e.g make it easier to run another version of the app in another region (although I doubt we'd ever do it?). I think it might not give us much (any?) benefit here. I think it's more that 'keep things region specific' is on the DevX best practice checklist so they highlighted it as we went along